source/installation/offline-environment.md

```diff
@@ -1,10 +1,10 @@
-# Installing OpenFisca in an offline environment
+# In an offline environment
 
 If you need to install OpenFisca on a server with no Internet access, here is how to do it.
 
 The big picture: download Python packages on a machine with Internet access, copy them to the server and install them in a [virtualenv](https://virtualenv.pypa.io/en/stable/).
 
-We assume that it is possible to copy files to the server, for example via an USB key. Or perhaps the server filters only outgoing connections, but accepts incoming connections allowing to copy the files.
```

Comment:
USB _stick_ in English, let's fix that! 😉

Reply:
Yes!